recod_frappe_devtools/build_docs/setup_docs.py

```diff
@@ -0,0 +1,368 @@
+
+        self.app_title = self.hooks.get("app_title")[0]
+        self.app_path = frappe.get_app_path(self.app)
+
+        print("Deleting current...")
+        shutil.rmtree(self.path, ignore_errors=True)
+        os.makedirs(self.path)
+
+        self.make_home_pages()
+
+        self.create_general_doc(frappe.get_app_path(self.target_app, 'www', 'docs'))
+
+        for basepath, folders, files in os.walk(self.app_path):
+
+            # make module home page
+            if "/doctype/" not in basepath and "doctype" in folders:
+                module = os.path.basename(basepath)
+
+                module_folder = os.path.join(self.models_base_path, module)
+
+                self.make_folder(module_folder,
+                                 template="templates/autodoc/module_home.html",
+                                 context={"name": module})
+                self.update_index_txt(module_folder)
+
+            # make for model files
+            if "/doctype/" in basepath:
+                parts = basepath.split("/")
+                # print parts
+                module, doctype = parts[-3], parts[-1]
+
+                if doctype != "boilerplate":
+                    self.write_model_file(basepath, module, doctype)
+
+            # standard python module
+            if self.is_py_module(basepath, folders, files):
+                self.write_modules(basepath, folders, files)
+
+        # self.build_user_docs()
+        self.copy_user_assets()
+        self.add_sidebars()
+        self.add_breadcrumbs_for_user_pages()
+
+    def add_breadcrumbs_for_user_pages(self):
+        for basepath, folders, files in os.walk(os.path.join(self.docs_path,
+                                                             'user')):  # pylint: disable=unused-variable
+            for fname in files:
+                if fname.endswith('.md') or fname.endswith('.html'):
+                    add_breadcrumbs_tag(os.path.join(basepath, fname))
+
+    def add_sidebars(self):
+        '''Add _sidebar.json in each folder in docs'''
+        for basepath, folders, files in os.walk(self.docs_path):  # pylint: disable=unused-variable
+            with open(os.path.join(basepath, '_sidebar.json'), 'w') as sidebarfile:
+                sidebarfile.write(frappe.as_json([
+                    {"title": "Search Docs ...", "type": "input", "route": "/search_docs"},
+                    {"title": "Docs Home", "route": "/docs"},
+                    {"title": "{} - User Guide".format(self.app_context['app'].get("title")),
+                     "route": "/docs/{}/user".format(self.target_app)},
+                    {"title": "{} - Server API".format(self.app_context['app'].get("title")),
+                     "route": "/docs/{}/current/api".format(self.target_app)},
+                    {"title": "{} - Models (Reference)".format(self.app_context['app'].get("title")),
+                     "route": "/docs/{}/current/models".format(self.target_app)},
```

Comment:
Issue already present in Frappe build-docs: use docs version for target path
```suggestion
                     "route": "/docs/{0}/{1}/models".format(self.target_app, self.app_context["app"].get("docs_version"))},
```